docs/userguide/src/en/bpmn/ch15-REST.adoc

```diff
@@ -3008,7 +3008,7 @@ POST runtime/tasks/{taskId}
 ----
 
 
-Completes the task. Optional variable array can be passed in using the +variables+ property. More information about the variable format can be found in <<restVariables,the REST variables section>>. Note that the variable-scope that is supplied is ignored and the variables are set on the parent-scope unless a variable exists in a local scope, which is overridden in this case. This is the same behavior as the +TaskService.completeTask(taskId, variables)+ invocation.
+Completes the task. Optional variable array can be passed in using the +variables+ property. More information about the variable format can be found in <<restVariables,the REST variables section>>. Note that the variable-scope is applied. If scope isn't indicated, variables are set to the primary scope and if indicated, variables are set to the scope that is provided. This is the same behavior as the +TaskService.completeTask(taskId, scopedVariableContainerHelper)+ invocation.
```

Comment:
Use "parent-scope" instead of primary scope.

Reply:
Ok, changed